modules/cli/src/main/scala/scala/cli/commands/CoursierOptions.scala

```diff
@@ -23,6 +24,7 @@ final case class CoursierOptions(
 }
 
 object CoursierOptions {
-  implicit val parser = Parser[CoursierOptions]
-  implicit val help   = Help[CoursierOptions]
+  implicit val parser                  = Parser[CoursierOptions]
+  implicit val help                    = Help[CoursierOptions]
+  implicit val rw: RW[CoursierOptions] = macroRW
```

Comment:
NIT: Let's call this… `jsonCodec` maybe? `rw` is too terse and doesn't really say what it is.
```suggestion
  implicit val jsonCodec: RW[CoursierOptions] = macroRW
```